### PR TITLE
[3.6.0] docs(pdfviewer): Initial PdfViewer Documentation

### PR DESCRIPTION
**Pull request by dimodi**

### CODE REVIEW

File: components/pdfviewer/overview.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+## Toolbar
+
+The [PdfViewer toolbar can render built-in and custom tools]({%slug pdfviewer-toolbar%}). The default tools are related to the built-in features such as:
+
+* Paging
+* Zoom
+* Text selection
+* Pan
+* Search
+* Open and download
```
[the remark this answers is not in the document]
Comment (dimodi) [reply components/pdfviewer/overview.md:63]: This is a list of features, not tools. I was even considering to merge text selection and panning to reduce the section space. If it looks strange, I can split everything.

[the remark this answers is not in the document]
Comment (dimodi) [reply components/pdfviewer/overview.md]: P.S. I grouped the features by topic in just three list items. Looks compact and consistent with the rest of the content.

File: components/pdfviewer/events.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+        if (file.Size > 1024 * 1024)
+        {
+            args.IsCancelled = true;
+            EventLog = $"Open cancelled conditionally. File {file.Name} ({file.Size} bytes) is larger than 1 MB.";
+        }
+        else
+        {
+            // Get the PDF file contents if necessary.
+            var buffer = new byte[file.Stream.Length];
```
Comment (ntacheva) [inline components/pdfviewer/events.md:108]: If we reach this point, the file is not opened. Is this expected?

**Review by ntacheva — APPROVED** [review]

File: components/pdfviewer/overview.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Parameter | Type and Default&nbsp;Value | Description |
+| --- | --- | --- |
+| `Class` | `string` | An additional CSS class for the `<div class="k-pdf-viewer">` element. Use it to [customize the component styles and override the theme]({%slug themes-override%}). |
+| `Data` | `byte[]` | The source of the currently displayed PDF file. |
+| `EnableLoaderContainer` | `bool` <br /> (`true`) | Determines if the PDF Viewer will show a loading animation during opening, downloading or zooming a PDF file. |
+| `Height` | `string` | The PdfViewer height as a [CSS length value]({%slug common-features/dimensions%}). If not set, the component will expand vertically, based on the loaded file. `Height` is required for the component paging and scrolling to work. |
+| `MaxZoom` | `decimal` <br /> (`4m`) | The largest possible zoom level. The default value is 400%. |
```
Comment (dimodi) [inline components/pdfviewer/overview.md:80]:
```suggestion
| `MaxZoom` | `decimal` <br /> (`4m`) | The largest possible zoom level. The default value allows zooming in 4 times (400%). |
```

File: components/pdfviewer/overview.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Parameter | Type and Default&nbsp;Value | Description |
+| --- | --- | --- |
+| `Class` | `string` | An additional CSS class for the `<div class="k-pdf-viewer">` element. Use it to [customize the component styles and override the theme]({%slug themes-override%}). |
+| `Data` | `byte[]` | The source of the currently displayed PDF file. |
+| `EnableLoaderContainer` | `bool` <br /> (`true`) | Determines if the PDF Viewer will show a loading animation during opening, downloading or zooming a PDF file. |
+| `Height` | `string` | The PdfViewer height as a [CSS length value]({%slug common-features/dimensions%}). If not set, the component will expand vertically, based on the loaded file. `Height` is required for the component paging and scrolling to work. |
+| `MaxZoom` | `decimal` <br /> (`4m`) | The largest possible zoom level. The default value allows zooming in 4 times (400%). |
+| `MinZoom` | `decimal` <br /> (`0.5m`) | The smallest possible zoom level. The default value is 50%. |
```
Comment (dimodi) [inline components/pdfviewer/overview.md:81]:
```suggestion
| `MinZoom` | `decimal` <br /> (`0.5m`) | The smallest possible zoom level. The default value allows zooming out to 50%. |
```

File: components/pdfviewer/events.md (hunk 2 above)
Comment (dimodi) [reply components/pdfviewer/events.md:108]: (SignalR message size)

**Pull request merged**